include/circt/Dialect/FIRRTL/FIRRTLAnnotationHelper.h

```diff
@@ -175,10 +175,13 @@ LogicalResult applyGCTDataTaps(AnnoPathValue target, DictionaryAttr anno,
 LogicalResult applyGCTMemTaps(AnnoPathValue target, DictionaryAttr anno,
                               ApplyState &state);
 
-/// Implements the same behavior as DictionaryAttr::getAs<A> to return the value
-/// of a specific type associated with a key in a dictionary.  However, this is
-/// specialized to print a useful error message, specific to custom annotation
-/// process, on failure.
+LogicalResult applyGCTSignalMappings(AnnoPathValue target, DictionaryAttr anno,
```

Comment:
Consider alpha-sorting these, unless there's significance in the current ordering (perhaps matches the table, idk).

Reply:
Can do that in a follow-up. I think it's currently matching the table, but only because the table is still in the order in which these were added.